scripts/verify-bundles.mjs

```diff
@@ -39,7 +39,10 @@ async function verifyBundles() {
       className: 'HotTable'
     }
   };
-  const { default: mainPackageJson } = await import('../package.json');
+  const { default: mainPackageJson } = await import('../package.json', {
+    assert: { type: 'json' }
+    }
```

Comment:
Weird formatting 🤔

Reply:
I agree. The linter forced me to do this 🙄 https://github.com/handsontable/handsontable/pull/10068/commits/1326ed95b00e1787d3945724f66525915d5ee2ba

Reply:
What about a one-liner? 
`const { default: mainPackageJson } = await import('../package.json', { assert: { type: 'json' }});`
or 
```js
const { 
  default: mainPackageJson
} = await import('../package.json', { assert: { type: 'json' }});
```